toolkit/components/uniffi-bindgen-gecko-js/src/templates/Scaffolding.cpp

```diff
@@ -30,7 +30,9 @@ struct Args {
 
 // Return values from the scaffolding function
 struct Result {
+    {%- if func.has_return_type() %}
```

Comment:
the compiler was complaining when functions don't have return types that the type is incomplete (since it would have a `void` member)